### ISSUE TITLE
Assess project color schemes in relation to accessibility 

### ISSUE THREAD

**Issue by eliotjordan** [geoblacklight/geoblacklight#1137]
HGL has done and analysis and integration of color combinations based on colorblindness. It would be worthwhile to see if there updates that we can make to core GeoBlacklight.

**Comment by eliotjordan** [comment]
Going to re-open this ticket to ask if there are other places in GeoBlacklight where the color combinations can be improved?

**Comment by mferrarini** [comment]
Update: discussed the metadata syntax colors and demo'ed to the group with a color-blindness simulator. Decided to leave as is with the red and green since there's more than just color distinguishing the different elements (i.e. the bolding and the shading).
